backend-shared/README.md

```diff
@@ -1,30 +1,65 @@
+# Managed GitOps Backend-Shared Libraries
 
-# Managed GitOps Backend Shared
+[![Go Report Card](https://goreportcard.com/badge/github.com/redhat-appstudio/managed-gitops/backend-shared)](https://goreportcard.com/report/github.com/redhat-appstudio/managed-gitops/backend-shared)
+[![GoDoc](https://godoc.org/github.com/redhat-appstudio/managed-gitops/backend-shared?status.svg)](https://pkg.go.dev/mod/github.com/redhat-appstudio/managed-gitops/backend-shared)
+[![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
```

Comment:
Can't we use this badges for the main README (parent repo) as well?

Reply:
some of them might work but the way this works is you point the relevant URLs into the relevant Go project. Yet I have no experience with monorepos so not sure about it. The main repo is not a Go project, so it would be tricky. I think the coverage badge works for the whole project because it it's setup for the whole project (?). If so we could add this in the main README and remove it from the others. As for all the others badges, most probably are not working outside in the main repo.

Reply:
Yes absolutely, what I selected for the monorepo is just to show godoc, license and the coverage badges (this is just for cosmetic purpose!).

Reply:
Godoc is not working in the parent repo :(